charts/rode/templates/deployment.yaml

```diff
@@ -42,23 +42,24 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default $.Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           args:
-            - "--port={{ .Values.container.port }}"
+            - "--http-port={{ .Values.container.httpPort }}"
+            - "--grpc-port={{ .Values.container.grpcPort }}"
             - "--grafeas-host={{ .Values.grafeasHost }}"
-            - "--opa-host={{ .Values.opa.host }}"
```

Comment:
We removed this flag since it's not in main, but in a [branch](https://github.com/rode/rode/blob/RODE-52-policy-attest/config/config.go#L63)